src/signals/incident-management/routes.js

```diff
@@ -9,6 +9,7 @@ const routes = {
   map: MAP_URL,
   incident: `${INCIDENT_URL}/:id(\\d+)`,
   split: `${INCIDENT_URL}/:id(\\d+)/split`,
+  splitIncident: `${INCIDENT_URL}/:id(\\d+)/splitincident`,
```

Comment:
One of these routes can go, right?

Reply:
Like discussed, it was unclear if the legacy one had to stay, done